gears/src/rest/error.rs

```diff
@@ -101,6 +101,7 @@ impl From<QueryError> for HTTPError {
             QueryError::Store(_) => {
                 HTTPError::not_found_with_msg("The requested version could not be found.".into())
             }
+            _ => HTTPError::bad_request("Invalid request.".to_owned()),
```

Comment:
We should try to be more helpful. Maybe add a TODO

Reply:
True

Reply:
Done